device/samples/javascript/test.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const Protocol = require('azure-iot-device-mqtt').MqttWs;
+const Client = require('azure-iot-device').Client;
+const Message = require('azure-iot-device').Message;
+const debug = require('debug');
+debug.enable('*');
+
+const deviceConnectionString = "HostName=vishnureddy-test-hub.azure-devices.net;DeviceId=vishnureddy-test;SharedAccessKey=DcpxkbpxNlG/Z8wEOEyZPpCIpTSwvj/MNHmaEE7elLo=";
```

Comment:
accidental check in. Deleted the hub resource, so this is no longer a valid connection string